src/goTest.ts

```diff
@@ -191,7 +191,11 @@ export function goTest(testconfig: TestConfig): Thenable<boolean> {
 		outputChannel.appendLine('');
 
 		let proc = cp.spawn(goRuntimePath, args, { env: testEnvVars, cwd: testconfig.dir });
-		proc.stdout.on('data', chunk => outputChannel.append(chunk.toString()));
+		proc.stdout.on('data', chunk => {
+			let testOutput = expandFilePathInOutput(chunk.toString(), testconfig.dir);
```

Comment:
Thanks @rakyll and @campoy 

The current needs isn't worth the trouble of changing the current behavior.

We will go with expanding the file name from the extensions's side